### PR TITLE
fix(refs T31070): fix accuracy in measure tools

### PR DESCRIPTION
**Pull request by salisdemos**
**Ticket:** https://yaits.demos-deutschland.de/T31070

Somehow we had a huge aberration between the map (and scale) and the measure tools. This was caused by using a openLayers method wrong. For Details see here: https://stackoverflow.com/questions/29548358/openlayers-ol3-linestring-getlength-not-returning-expected-value

So the main change here is using methods from `ol/sphere` instead of `getLength` from the measuretool itself

While investigating the behavior, I had to clean up the code a bit and moved Methods. (Sorry for making the PR harder to review)

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/map/publicdetail/Map.vue [hunk 1]
```diff
@@ -299,11 +305,43 @@ export default {
...
+
+        this.scope = territoryLayer
+
+        //  Add behavior to button inside map.
+        this.addCustomLayerToggleButton({
+          id: 'territorySwitcher',
+          layerName: 'territory',
+          activated: true
+        })
+      }
```
Comment (salisdemos) [inline client/js/components/map/publicdetail/Map.vue:344]: this is just moved to a dedicated method

**Review by hwiem — APPROVED** [review]
lgtm + works, just nitpicks

File: client/js/components/map/publicdetail/Map.vue [hunk 2]
```diff
@@ -1518,16 +1490,16 @@ export default {
         //  Unselect tools
         $(this.prefixClass('.c-map__tool, .c-map__tool-simple')).removeClass(this.prefixClass('is-active'))
 
-        resetMeasureOutput()
+        // resetMeasureOutput()
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply client/js/components/map/publicdetail/Map.vue:1493]: Yes. This removed an Eventlistener which is not used anymore and recreated the measureTooltip wich is not (it it looks like it wasn't before) neccessary. (or I tested wrong)

[the remark this answers is not in the document]
Comment (salisdemos) [reply client/js/components/map/publicdetail/Map.vue]: removed in 0dec843214cb6eef147ffa4796742d604017640e

File: client/js/components/map/publicdetail/Map.vue [hunk 3]
```diff
@@ -1414,26 +1384,27 @@ export default {
       measureTools.forEach(measureTool => {
         const measure = drawInteraction(measureSource, measureTool.interaction)
         let doubleClickListener
+        let sketch
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply client/js/components/map/publicdetail/Map.vue]: ok. You wanted it like so 54554d5cf5e7887c08a9c2292ead11eb05ed95b8

**Review by hwiem — APPROVED** [review]

**Pull request merged**